src/mixins/filing-template-mixin.ts

```diff
@@ -432,98 +436,90 @@ export default class FilingTemplateMixin extends Vue {
 
     this.setProvisionsRemoved(filing.alteration.provisionsRemoved)
 
-    // Set Office Addresses
-    this.setOfficeAddresses(businessSnapshot[2])
+    // Store office addresses
+    // *** TODO: should restore office addresses from alteration, not snapshot???
+    this.setOfficeAddresses(businessSnapshot.incorporationAddress)
```

Comment:
Who was working on office addresses? Please see TODO comment (and please respond here).